examples/groovy/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
```

Comment:
Why did you need to add gradle wrapper to both examples? It could be run from project root like `./gradlew -p examples/groovy dependencyUpdates` and we could keep single wrapper instead of 3.

Reply:
Oh, very neat, I did that and documented in the README